ExceptionDemo.assignment2('jeffeinstienber');


ExceptionDemo.demo3(null, new list<integer>{10,5});
ExceptionDemo.demo3(null, new list<integer>{10,0});
ExceptionDemo.demo3('null', new list<integer>{10,0});
ExceptionDemo.demo3('null', new list<integer>{10});
ExceptionDemo.demo3('null', new list<integer>());



ExceptionDemo.demo2(null); //null pointer exception
ExceptionDemo.demo2('banana');//dml exceptions

ExceptionDemo.demo1(null);

account a11 = ExceptionDemo.demo2('banana');
system.debug(a11);

 ExceptionDemo.demo2(null);




 
